shapeshift/web
# feat: fox page, get fox wire up

## Description

This implements the wire up of "Get Fox" component into the fox page.
Also FOXy description is present everywhere we have description even with Fox page flag off.

## Notice




## Pull Request Type


## Issue (if applicable)

https://github.com/shapeshift/web/issues/1778


## Risk

No risk


## Testing

- Navigate to fox token page
- Click on Buy FOX => should open new page to coinbase for getting fox token
- Click on Receive for FOX token => should open the modal for receiving
- Click on FOXy tab
- Click on Get for FOXy token => should open the FOXy staking modal
- Click on Receive for FOXy => should open the modal for receiving
- Navigate to FOXy asset page
- Verify FOXy description is present



## Screenshots (if applicable)

![fox](https://user-images.githubusercontent.com/5720927/169775389-53581404-5b6f-4a36-a23c-a6a04198b9e1.png)

![foxy](https://user-images.githubusercontent.com/5720927/169775400-b57a9b73-3617-4219-9f87-3730635846bc.png)
